web/themes/joinup/prototype/sass/components/_icon.sass

```diff
@@ -155,6 +155,9 @@ $waat-icons-font-path: '../fonts/waat-icons/waat-icons'
 .icon--github:before
   content: '\e8d6'
 
+.icon--star:before
```

Comment:
Missing inclusion on the top of the file which will help on prototype demo.

Reply:
New icons have been added to the styleguide.